collect_app/src/main/java/org/odk/collect/android/storage/StoragePathProvider.java

```diff
@@ -79,11 +79,11 @@ public String getStorageRootDirPath() {
                 : getUnscopedStorageRootDirPath();
     }
 
-    public String getTmpFilePath() {
+    public String getTmpImageFilePath() {
```

Comment:
This is an existing issue but still want to highlight it. I don't think we know the image is a `jpg`. It could be a `png`, `bmp`, ... We should fix this at some point. Maybe a separate issue?

Reply:
You are right I'll file an issue.

Reply:
Hmm unfortunately I think it's not possible... we pass our tmp file starting a camera app because it's a way to capture a full-size photo:
https://developer.android.com/training/camera/photobasics#TaskPath

we don't know what format such an external app is going to return so like in the documentation we use jpg. Then the captured photo is saved to our tmp file.

Reply:
I see. Can we check the mime type and fix the extension later? I think all images are submitted with a jpg extension but maybe I’m remembering wrong.

Reply:
No really, I mean we can read mime type using the tmp file where an external app saves captured photos but it's pointless because it will always be jpeg.